.travis.yml

```diff
@@ -38,11 +42,18 @@ script:
 jobs:
   fast_finish: true
   include:
-    - stage: test_changed
+    - stage: test_pr
       script:
-        # FIXME: re-enable as soon as we fix the manifest files
-        # - ddev -e manifest verify --include-extras
+        - ddev validate manifest -i
+        - ddev validate config
+        - ddev validate service-checks
+        - ddev validate metadata
+        # with no params, ddev will only test integrations that are part of the branch diff
         - ddev -e test
+    - stage: test
+      env: CHECK=aqua
+    - stage: test
+      env: CHECK=eventstore
```

Comment:
Why are all the others not here ?

Reply:
Need to be migrated to wheels 🙈

Reply:
oh right I forgot about that. LGTM then.